[PPV2] Update platforms support

Wilfrid-Unity opened the pull request:
### Purpose of this PR
This PR adds PPV2 support for Shader define of upcoming platform.

---
### Testing status
UTJ QA team verified these PPV2 package changes as part of validation of Unity version used by pre-launch titles, using test project "Shinai" (see report [here](https://docs.google.com/spreadsheets/d/18hkAvNKsIt1zTRku4UL0CsE18xgUdtjNaN4YdNnOVzo/edit?gid=1081259977#gid=1081259977) ).



---
### Comments to reviewers
Landing of changes approved in this [thread](https://unity.slack.com/archives/C07102MAB/p1748961664010479)
This is part of worked tracked by these tickets: [PLAT-13976](https://jira.unity3d.com/browse/PLAT-13976), [PLAT-13939](https://jira.unity3d.com/browse/PLAT-13939)

BenGraterUnity reviewed (changes requested):
@Wilfrid-Unity Thanks for putting this together!

Looks good, although I'm not sure if/how I am able to test this currently.  I have provided a couple *minor* comments.

Could you please expand the PR description on how specifically this was tested in editor and device?
Also, maybe just scan the [package documentation](https://docs.unity3d.com/Packages/com.unity.postprocessing@latest/) to make sure there's nothing that would need updating there (probably not).

Wilfrid-Unity commented on com.unity.postprocessing/CHANGELOG.md:
```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
-- 
+- Updated Platforms support
```
It's fine to mention ; I updated the message in d74e354

Wilfrid-Unity commented:
> I have provided a couple minor comments. 

> Could you please expand the PR description on how specifically this was tested in editor and device?
Also, maybe just scan the [package documentation](https://docs.unity3d.com/Packages/com.unity.postprocessing@latest/) to make sure there's nothing that would need updating there (probably not).

Thanks @BenGraterUnity, I have addressed your comments in commits 6cabdca..d74e354
I checked the package documentation and spotted nothing platform-specific.
I also added info in the PR about the testing these changes underwent ; let me know if you want more details.

BenGraterUnity commented:
@theopnv Could you merge this please?
